cafebabel/templates/articles/macros.html

```diff
@@ -20,13 +20,16 @@
   <div class=tags>
     <label for=tag-1>Tags</label>
     <div class=tag-container><input name=tag-new><button class=add>+</button></div>
-    {% if article and article.tags %}
-    {% set count = 1 %}
+    <ul class=tagsList>
```

Comment:
We haven't used camelCased class names so far but hyphen separated.
Let's try to be consistent